modin/pandas/test/test_io.py

```diff
@@ -428,6 +452,10 @@ def test_read_csv_squeeze(self, test_case):
         )
 
     def test_read_csv_mangle_dupe_cols(self):
+        if Backend.get() == "Omnisci":
+            pytest.xfail(
+                "processing of duplicated columns in OmniSci backend is not supported yet"
```

Comment:
submit a tracking issue and reference it here

Reply:
Fixed.